spec/requests/api/v0/tags_spec.rb

```diff
@@ -52,4 +52,31 @@
       expect(response.headers["surrogate-key"].split.to_set).to eq(expected_key)
     end
   end
+
+  describe "GET /api/tags/me" do
+    let(:tags) { create_list(:tag, 5) }
+    let(:user) { create(:user) }
+    let(:api_token) { create(:api_secret, user: user) }
+
+    before do
+      followed_tags = tags.first(3)
+      followed_tags.each { |tag| user.follow(tag) }
+    end
+
+    context "with invalid authentication" do
+      it "returns unauthorized" do
+        get me_api_tags_path, headers: { "api-key" => "askjdfhlk" }
+
+        expect(response).to have_http_status(:unauthorized)
+      end
+    end
+
+    context "with valid authentication" do
```

Comment:
This spec fails with:
```
  1) Api::V0::Tags GET /api/tags/me with valid authentication returns only the tags followed by the user
     Failure/Error: get me_api_tags_path, headers: { "api-key" => api_token.secret }

     Bullet::Notification::UnoptimizedQueryError:
       user: YouTube
       GET /api/tags/me
       AVOID eager loading detected
         Follow => [:followable]
         Remove from your query: .includes([:followable])
       Call stack
         /Users/YouTube/Development/dev.to/spec/requests/api/v0/tags_spec.rb:76:in `block (4 levels) in <top (required)>'
         /Users/YouTube/Development/dev.to/spec/rails_helper.rb:135:in `block (3 levels) in <top (required)>'
         /Users/YouTube/Development/dev.to/spec/rails_helper.rb:135:in `block (2 levels) in <top (required)>'
```
I removed the `includes([:followable])` from the `TagsController` and the error did not change or go away, so I added it back. @rhymes Do you know what this could be about? I couldn't figure it out.

Reply:
My guess is the `follows_by_type` scope might be including this. We can disable it, @rhymes would probably have the best thoughts on how to do that.

Reply:
The main issue here is that you're not rendering `Tag` objects but `Follow` objects but then you're not accessing the `followable` object, see my comment in the controller